lib/gui.js

```diff
@@ -67,9 +67,9 @@ define(['d3-interpolate', 'map', 'sidebar', 'tabs', 'container', 'legend',
       buttonToggle.setAttribute('data-tooltip', _.t('button.switchView'));
       buttonToggle.onclick = function onclick() {
         if (content.constructor === Map) {
-          router.view('g');
+          router.fullUrl({ view: 'graph', lat: undefined });
```

Comment:
sollte es hierfür nicht ein `if (data.view === 'graph') { data.lat = undefined;}` in fullUrl des routers geben?